tests/features/webtoolsAnalytics.feature

```diff
@@ -0,0 +1,30 @@
+@api @webtoolsAnalytics
+Feature: Check Piwik
+  In order to check if the type attribute is set for the Piwik element.
```

Comment:
In order to have an insight in the usage of my website
As a site owner
I want to run analytics on my site

Reply:
But we don't have a specific requirement for the site owner, so I would suggest to let it cover wider user range at the moment, then when we know about the roles and permission we can restrict it.

Reply:
Ah, well I was trying to rephrase it reflect more business value. The site owner is not a role in the sense of a Drupal role.